stac_geoparquet/arrow/_delta_lake.py

```diff
@@ -21,14 +21,35 @@ def parse_stac_ndjson_to_delta_lake(
     chunk_size: int = 65536,
     schema: pa.Schema | None = None,
     limit: int | None = None,
+    schema_version: Literal["1.0.0", "1.1.0"] = "1.0.0",
     **kwargs: Any,
 ) -> None:
+    """Convert one or more newline-delimited JSON STAC files to Delta Lake
+
+    Args:
+        input_path: One or more paths to files with STAC items.
+        table_or_uri: A path to the output Delta Lake table
+
+    Args:
+        chunk_size: The chunk size to use for reading JSON into memory. Defaults to
+            65536.
+        schema: The schema to represent the input STAC data. Defaults to None, in which
+            case the schema will first be inferred via a full pass over the input data.
+            In this case, there will be two full passes over the input data: one to
+            infer a common schema across all data and another to read the data and
+            iteratively convert to GeoParquet.
+        limit: The maximum number of JSON records to convert.
+        schema_version: GeoParquet specification version; if not provided will default
+            to latest supported version.
```

Comment:
Seems to be `1.0.0` right now. Does that mean "1.1.0" isn't supported, or am I misreading it.

Reply:
I changed it back to 1.0 because the 1.1 spec hasn't actually been released yet, and I think we were writing `1.1-dev` previously. Hopefully 1.1 will be released soon (today?) and then we can ensure we're compatible with 1.1 and change back to 1.1 being the default

Reply:
1.1 was released yesterday so now we can just make 1.1 the default